### ISSUE TITLE
Mura 7 PostgreSQL Table Error for Files

### ISSUE THREAD

**Issue by dbudde** [blueriver/MuraCMS#2259]
I just completed a fresh Mura 7 install and was going to upload a file under an already created page using the Add Content > File.  I got the attached error for the PostgreSQL table.  Changing the 'filesize' column from 'integer' to 'character varying (25)' fixed the issue.  I am not sure if that is the proper fix or not.

[error_filesize.pdf](https://github.com/blueriver/MuraCMS/files/252941/error_filesize.pdf)

**Comment by cfexpert** [comment]
Mura 7.0.6749. Same PostgreSQL Problem still exists when uploading an associated image. No such error when using different database engine. Using the same fix, but if feels a little bit 'hacky'.

**Comment by mattlevine** [comment]
I'll look into it

**Comment by mattlevine** [comment]
Can you post some example values that are being save to the filesize column? Are they strings that just happen to be numbers?

**Comment by mattlevine** [comment]
Please update your core and retest.

**Comment by cfexpert** [comment]
Thank you very much for your quick response. It's highly appreciated. Unfortunately the fix didn't help in this very case. I did some further investigations and I found out, that on Ubuntu 14.04.5 LTS, PostgreSQL 9.3.14, Lucee 5 the Problem does not exist, not even in Mura 7.0.6730 .

On Windows 2012, ColdFusion 11, PostgreSQL 9.5.0 the error is thrown on uploading the associated image and adding content of type file. The Query Parameters are looking fine, tho. fileSize = 329253

Maybe we should try a different database jdbc driver.

[queryparameters.txt](https://github.com/blueriver/MuraCMS/files/489127/queryparameters.txt)
